MvvmCross/Platform/Platform/ReflectionExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static IEnumerable<ConstructorInfo> GetConstructors(this Type type)
 
         public static bool IsInstanceOfType(this Type type, object obj)
         {
-            return type.IsAssignableFrom(obj.GetType()) || obj.IsMarshalByRefObject();
+            return obj != null && (type.IsAssignableFrom(obj.GetType()) || obj.IsMarshalByRefObject());
```

Comment:
This change isn't mentioned in the PR description, can you elaborate on it?

Reply:
This was just a minor change - if this method is accidentally called with obj == null, it'll throw an exception. 
The fix, allows this method to be called passing in null

Reply:
I think this should be in a separate PR.